### ISSUE TITLE
[Bug] Gitflow-Finish Feature fails when using GPG commits

### ISSUE THREAD

**Issue by acanewby** [ForkIssues/Tracker#1936]
When I use Gitflow and non-signed commits, all Gitflow features work correctly.  However, as soon as I try and finish a Feature with GPG-signed commits, the following failures occur:

1. Spurious merge conflicts are reported
2. When the error dialog is closed, merge conflicts are reported as fixed, requiring a further commit onto `origin/develop`
3. Following the commit,  the feature branch still remains and must be deleted manually

Expected behavior:  same experience as when using unsigned commits e.g.

1. Merge completes without error
2. Feature branch is deleted

I also experience similar behavior with Gitflow-Finish Release, which I assume uses the same underlying mechanism.

Happy to participate in error reproduction, diagnosis and testing.

<img width="1455" alt="screen-1" src="https://github.com/fork-dev/Tracker/assets/12296025/3f9298fb-9a49-4cdb-8408-f4f03ce7cf9f">

<img width="1457" alt="screen-2" src="https://github.com/fork-dev/Tracker/assets/12296025/13bf9f15-9565-47d4-91ac-35b448083533">

<img width="1463" alt="screen-3" src="https://github.com/fork-dev/Tracker/assets/12296025/285b7d86-fda1-4fbe-9773-3c62b3cc3d52">

<img width="1470" alt="screen-4" src="https://github.com/fork-dev/Tracker/assets/12296025/866b0f56-75f1-41bc-bd45-abf20ed9afc7">

**Comment by DanPristupov** [comment]
Gitflow is deprecated and not supported.

I don't know what's the problem and most likely won't spend time investigating the problem. I'd recommend to simply merge the commit into the develop branch.

**Comment by acanewby** [comment]
Thanks for the quick reply.

I see from the Release Notes that fast-forward was added to git-flow in 2.4 but nothing since then about deprecation so hard for users to understand what features have a future. 

Is there a public list of deprecated features?

**Comment by DanPristupov** [comment]
Gitflow is project on GitHub. It's not related to Fork.

https://github.com/petervanderdoes/gitflow-avh

**Comment by acanewby** [comment]
Yes - perhaps I should have been clearer.

As the repo says:  "A collection of Git extensions to provide high-level repository operations for Vincent Driessen's [branching model](http://nvie.com/git-model)."

I am not really interested in the gitHub-avh project which, as you say, is now archived.

I'm interested in Fork's continued support for the Vincent Driessen branching model, which is a concept independent of petervanderdoes' solution. 

gitflow as a concept is widely supported across the industry (e.g. IntelliJ IDEA, SmartGit, GitKraken etc. etc.)

I was simply hoping that Fork could continue its great support for that branching model.  I would be surprised to hear that it was in any way dependent on gitflow-avh's shell script approach, since Fork appears to be Swift-based.

**Comment by DanPristupov** [comment]
> I would be surprised to hear that it [Fork] was in any way dependent on gitflow-avh's shell script approach, since Fork appears to be Swift-based.

Fork just wraps git-flow-avh scripts. It doesn't implement any of git-flow functionality.

**Comment by acanewby** [comment]
Ok - makes total sense.  

Would you be interested if I forked the GitFlow-AVH repo and fixed the issue?

Would that require access to a dev build of Fork?

**Comment by DanPristupov** [comment]
Git-flow-avh is a part of the official git-for-windows distribution. You will also need to take the responsibility for it. I mean, it's not as easy as it sounds.

**Comment by acanewby** [comment]
Yeah.  Not going to do that.  It's up to git-for-windows how it wants to deal with Git-flow-avh deprecation.  Presumably, it will have to do something sooner or later.

I'm only interested in helping Fork stay my git client of choice, which means having a working gitflow.

Up for a chat if you can suggest a Fork-only approach.  If not, then I will figure out what else I can do.

I do see you have bundled the git-flow-avh scripts into the Fork .app under Resources/git-flow-avh. If I symlink that location to a fork of git-flow-avh, I guess I could mess around and at least figure out where things are breaking.  That should work right?
